implementation-details.md

```diff
@@ -431,9 +434,7 @@ The gossip loop is an infinite loop executed by a node in a separate thread. In
 
 ## TODOs
 
-* _how often the nodes should send ping messages_
 * _how much time the node has to reply to a ping message_
 * _how much time the node has to reply to pull request_
 * _what happens if the node ignores the prune message and sends the push message anyway to everybody_
```

Comment:
We know the answers to these questions - for all of them we can say _nothing happens_, I'm not sure though whether we should post them here...